common/org.eclipse.tracecompass.incubator.target/tracecompass-incubator-master.target

```diff
@@ -131,7 +131,7 @@
 <unit id="org.eclipse.tracecompass.lttng2.control.feature.group" version="0.0.0"/>
 <unit id="org.eclipse.tracecompass.jsontrace.feature.group" version="0.0.0"/>
 <unit id="org.eclipse.tracecompass.tmf.cli.feature.group" version="0.0.0"/>
-<repository location="https://download.eclipse.org/tracecompass/master/repository/"/>
+<repository location="https://download.eclipse.org/tracecompass/releases/9.3.0/repository/"/>
```

Comment:
Does this mean all stable branches from incubator must diverge from master branch from now on? They haven't usually.

Reply:
all stable branches don't build because they reference tc master. We would have to change the targets in ALL stable branches. But I think it's overkill. We can do it on need base.